### ISSUE TITLE
Gallery example 'Using Polar and Log-Polar Transformations for Registration' needs fixing

### ISSUE THREAD

**Issue by mellertd** [scikit-image/scikit-image#6304]
## Description
The gallery example for Using Polar and Log-Polar Transformations for Registration seems to be broken as of 0.19.x. Was working in 0.18.x.

## Way to reproduce
Unsure if this is specific to the production doc build or an issue in the code. I am raising this issue to get it on the board before starting my testing.

## Version information
See description. I'll report back in this thread when I know more.

**Comment by mellertd** [comment]
The `phase_cross_correlation` normalization parameter seems to be causing the trouble here. This parameter defaults to "phase" in 0.19.2. When I set to `None`, values seem to match 0.18.3 behavior. I'll do some more rigorous testing and if this holds up, will submit a PR with the fix.

**Comment by grlee77** [comment]
I think we had considered that change a bug fix, so we didn't have the usual deprecation cycle. Fortunately, we did keep the option to use the old behavior, so it is likely the fix will just require updating the demo.

**Comment by grlee77** [comment]
found the prior discussion: https://github.com/scikit-image/scikit-image/pull/5461#issuecomment-907785652

**Comment by mellertd** [comment]
@grlee77 My plan was to just drop `normalization=None` in all of the relevant positions in that gallery example, but I wanted to understand what was going on a little better first. I can't quite understand why the approach wouldn't be at least somewhat robust to that normalization, but maybe it's because the starting image is itself polar-log transformed frequency space? It's making my head hurt a bit to think through.

I'll update the demo and submit.

**Comment by grlee77** [comment]
Yeah, I agree it would be better to understand a bit better. I don't have a good intuition for it, but perhaps some debugging looking at intermediate results within the `phase_cross_correlation` function can provide some idea.

**Comment by mellertd** [comment]
straightforward fix is in the PR noted above. Still don't really understand why there is a big difference but that investigation will have to wait. At least the gallery example is working again

**Comment by jmhb0** [comment]
FYI: the web page for this still seems to have the broken code (though the fix said above solved it). 
https://scikit-image.org/docs/stable/auto_examples/registration/plot_register_rotation.html

**Comment by grlee77** [comment]
> FYI: the web page for this still seems to have the broken code (though the fix said above solved it). 
> https://scikit-image.org/docs/stable/auto_examples/registration/plot_register_rotation.html
> 


Most likely just need to a select the development version of the docs via the "All Versions" link at the left.
